dmutils/dates.py

```diff
@@ -0,0 +1,54 @@
+from six import string_types
+from datetime import datetime, timedelta
+from workdays import workday
+
+
+def get_publishing_dates(brief):
+    APPLICATION_OPEN_DAYS = {'1 week': 7, '2 weeks': 14}
+    QUESTIONS_OPEN_DAYS = {'1 week': 2, '2 weeks': 5}
+    dates = {}
+
+    if _brief_is_from_api_(brief):
+        dates['published_date'] = _get_start_date_from_brief_api(brief)
+    elif _brief_is_from_frontend_app_and_published_(brief):
+        dates['published_date'] = _get_start_date_from_published_frontend_brief(brief)
+    else:
+        dates['published_date'] = _get_todays_date()
+
+    length = _get_length_of_brief(brief)
+
+    dates['closing_date'] = dates['published_date'] + timedelta(days=APPLICATION_OPEN_DAYS[length])
+    dates['questions_close'] = workday(dates['published_date'], QUESTIONS_OPEN_DAYS[length])
+    dates['answers_close'] = workday(dates['closing_date'], -1)
+    dates['application_open_weeks'] = APPLICATION_OPEN_DAYS[length]//7
```

Comment:
It seems weird to keep this key in just for the purposes [of the template in the buyer frontend app](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/7e6fdb56610df1b5d270b136ad3dcc08569cba9f/app/templates/buyers/brief_publish_confirmation.html#L48), when it's just recreating what our variable is already called.

If we return it in the serialise method, we can just use it directly in the template.